### PR TITLE
Wip location endpoints

### PR DESCRIPTION
**Pull request by jafow**
don't forget to `npm install`

### CODE REVIEW

**Comment by jafow** [comment]
opening this for now as a placeholder

File: backend/controllers/jurisdictions.js [hunk 1]
```diff
@@ -34,11 +35,17 @@ exports.delete = (req, res, next) => {
  * list wip jurisidictions
  */
 exports.listWip = async (req, res) => {
-  let route = req.route.path
-  logger.info({ message: 'List wip jurisdictions', route: route })
+  let editorUserId = req.query.editor
```
Comment (jafow) [inline backend/controllers/jurisdictions.js:38]: allows us to `axios.get('/jurisdiction/wip?editor=2` to get all the wip jurisdcitions last touched by editor id 2

File: backend/routes/jurisdictions.js [hunk 2]
```diff
@@ -1,6 +1,9 @@
+const { checkSchema } = require('express-validator')
+
```
Comment (jmensch1) [inline backend/routes/jurisdictions.js:2]: ooh, nice

File: backend/controllers/locations.js [hunk 3]
```diff
@@ -17,18 +20,103 @@ exports.delete = (req, res, next) => {
...
+        wipJurisdictionId: wipJurisdictionId,
+      },
+      { where: { id: wipLocationId } }
+    )
+    logger.info({
+      message: 'Success: Updated wip location',
+      wipJurisdictionId: wipJurisdictionId,
+      wipLocationId: wipLocationId,
+      editorUserId: editorUserId,
+    })
```
Comment (jmensch1) [inline backend/controllers/locations.js:117]: fyi if the name of the property is the same as the name of the variable, you only need to type it once:
```
    logger.info({
      message: 'Success: Updated wip location',
      wipJurisdictionId,
      wipLocationId,
      editorUserId,
    })

**Review by jmensch1 — APPROVED** [review]
looks good

**Pull request merged**